src/components/apps/utils.js

```diff
@@ -31,11 +32,22 @@ export const getListingPath = (
     page,
     rowsPerPage,
     filter,
-    category
+    category,
+    isAdmin
 ) => {
-    const encodedFilter = encodeURIComponent(filter);
-    const encodedCategory = encodeURIComponent(category);
-    return `/${NavigationConstants.APPS}?selectedOrder=${order}&selectedOrderBy=${orderBy}&selectedPage=${page}&selectedRowsPerPage=${rowsPerPage}&selectedFilter=${encodedFilter}&selectedCategory=${encodedCategory}`;
+    return {
+        pathname: isAdmin
```

Comment:
Something to consider, which is really minor, but it might be a little clearer if instead of `isAdmin`, the variable that we're using throughout these changes were `isAdminView` or something similar.   When reading through this PR, I initially got the impression that we were disabling features for administrators, rather than reusing a component for user vs. admin view.  Just a suggestion though, feel free to ignore.

Reply:
Good point although am little torn here. The variable `isAdmin`  is derived from checking the `userProfile` to see if the user belongs to admin group. May it could be copied into `isAdminView` when sending it down to `utils` and `Toolbar` files 🤔 ?. Let me know @ash3rz @psarando

Reply:
I think I see the confusion now. If you don't see the code in `pages/admin/apps.js` first, then it might seem like there is only 1 app listing in our app that displays differently (or disables features) for admin users.

So I think `const isAdmin = profile?.admin` makes sense in `pages/admin/apps.js`, but it might clear up the confusion if `isAdminView` (or even just `adminView`) is used as the prop in the `Listing` on down:
```jsx
            <Listing
                baseId="apps"
                ...
                adminView={isAdmin}
            />
```

Reply:
Yup, that would clear it up IMO

Reply:
Ok this should be done.